wiki/Internet_Relay_Chat/id.md

```diff
@@ -73,8 +74,9 @@ Jika klien yang anda pakai tidak tercantum di sini, ikuti dokumentasi yang terse
 
 Tidak. Anda hanya dapat menggunakan nama pengguna osu! anda.
 
-### Apa fungsi status suara yang saya miliki ini? Saya juga melihat beberapa orang memilikinya
+### Mengapa beberapa nama pengguna diawali dengan tanda yang berbeda ?
 
-Semua pengguna dengan *status suara* terhubung menggunakan klien IRC, kecuali moderator chat yang selalu memiliki status *operator (+o)* terlepas dari klien apapun yang mereka gunakan.
+Konsep IRC pada umumnya memiliki beberapa mode kanal atau sekumpulan aksi di mana setiap pengguna dapat menggunakanya. Di osu!Bancho terdapat dua mode yang digunakan untuk mewakilkan kelompok khusus pengguna obrolan:
 
-Pengguna yang tidak memiliki status, terhubung menggunakan klien osu!.
+- `+`, atau "voice status": pengguna terhubung dengan klien IRC eksternal
+- `@`, or "chat operator status": pengguna adalah seorang moderator ([GMT](/wiki/GMT) or [NAT](/wiki/NAT))
```

Comment:
```suggestion
- `@`, or "chat operator status": pengguna adalah seorang chat moderator ([GMT](/wiki/GMT) atau [NAT](/wiki/NAT))
```

Reply:
the group of gmt has `chat moderator` and just `moderator`.

Reply:
shouldn't be necessary to separate those.

Chat moderator and "Moderator" is essentially the same thing (/group), which is GMT.